lib/bibliothecary/parsers/conda.rb

```diff
@@ -37,6 +41,26 @@ def self.parse_conda(file_contents)
 
         JSON.parse(response.body)
       end
+
+      def self.analyse_contents_from_info(info)
+        manifest = parse_conda(info.contents)
+        [
+          Bibliothecary::Analyser.create_analysis(
+            "conda",
+            info.relative_path,
+            "manifest",
+            map_dependencies(manifest, "manifest", "runtime")
+          ),
+          Bibliothecary::Analyser.create_analysis(
+            "conda",
+            info.relative_path,
+            "lockfile",
+            map_dependencies(manifest, "lockfile", "runtime")
+          )
+        ]
+      rescue Bibliothecary::FileParsingError => e
```

Comment:
Looks like this is no longer thrown here (it's in `parse_file` which isn't being called for conda)

Reply:
Should you catch the `RemoteParsingError` instead and create an error analysis from that?

Reply:
ah yeah good call thx guys